express/scripts/scripts.js

```diff
@@ -737,6 +743,25 @@ export function getMetadata(name) {
   return $meta && $meta.content;
 }
 
+/**
+ * fetches the string variables.
+ * @returns {object} localized variables
+ */
+
+export async function fetchPlaceholders() {
+  if (!window.placeholders) {
+    const locale = getLocale(window.location);
+    const urlPrefix = locale === 'us' ? '' : `/${locale}`;
+    const resp = await fetch(`${urlPrefix}/express/placeholders.json`);
+    const json = await resp.json();
```

Comment:
I think we should guard for `404s` here (and potentially fall back to english in this case?)

Reply:
i _think_ this may be resolved, but would appreciate your feedback.